barretenberg/cpp/src/barretenberg/crypto/pedersen_commitment/c_bind_new.cpp

```diff
@@ -50,7 +40,10 @@ WASM_EXPORT void pedersen___compress_with_hash_index(fr::vec_in_buf inputs_buffe
 {
     std::vector<grumpkin::fq> to_compress;
     read(inputs_buffer, to_compress);
-    auto r = crypto::pedersen_commitment::compress_native(to_compress, ntohl(*hash_index));
+    const size_t generator_offset = ntohl(*hash_index);
+    crypto::GeneratorContext<curve::Grumpkin> ctx; // todo fix
```

Comment:
Note: TODO -- not really sure what this fix is referring to

Reply:
Ah I thought I got rid of that. There's nothing to fix here I just wasn't happy with the interface